dashboard/app.R

```diff
@@ -582,6 +732,85 @@ server <- function(input, output, session) {
     }
   })
 
+  updateAsOfData = function() {
+    dataSource = "jhu-csse"
+    if(input$targetVariable == "Cases") {
+      targetSignal = "confirmed_incidence_num"
+    } else if (input$targetVariable == "Deaths") {
+      targetSignal = "deaths_incidence_num"
+    } else if (input$targetVariable == "Hospitalizations") {
+      targetSignal = "confirmed_admissions_covid_1d"
```

Comment:
@brookslogan just double checking that HUB is forecasting confirmed adult and pediatric admissions (=the signal used here), and not confirmed+suspected

Reply:
Will confirm in the slack channel. Merging to dev for now, can change the signal if needed later.